Add combined cf dashboard

mtekel opened the pull request:
Add the dashboard we show next to our CI and PR screen. It has combined
stats from CF and DEA dashboards. The graphs are scaled to have a good
fit on the full HD TV screen.

mtekel committed:
Add combined cf dashboard

Add the dashboard we show next to our CI and PR screen. It has combined
stats from CF and DEA dashboards. The graphs are scaled to have a good
fit on the full HD TV screen.

actionjack commented:
Hi Michael any reason why this has the same dashboard name as https://github.com/alphagov/cf-terraform/blob/combined-dashboard/scripts/grafana_dashboards/cf_services_metrics.json#L3 ?

mtekel committed:
Fix combined dashboard name

This dashboard was originally created by extending CF Metrics dashboard
and it kept using that name. Without name change, when applying changes
to grafana, dashboards with same name get overwritten by each other.

mtekel commented:
Fixed. Thanks for spotting.
